orchagent/portsorch.cpp

```diff
@@ -2312,6 +2312,27 @@ bool PortsOrch::addHostIntfs(Port &port, string alias, sai_object_id_t &host_int
     return true;
 }
 
+bool PortsOrch::setBridgePortLearningFDB(Port &port, bool enable)
```

Comment:
Probably this should be a more generic function for setting FDB learning mode with something like
bool PortsOrch::setBridgePortLearningFDB(Port &port,  sai_bridge_port_fdb_learning_mode_t mode)

```
typedef enum _sai_bridge_port_fdb_learning_mode_t
{
    /** Drop packets with unknown source MAC. Do not learn. Do not forward */
    SAI_BRIDGE_PORT_FDB_LEARNING_MODE_DROP,

    /** Do not learn unknown source MAC. Forward based on destination MAC */
    SAI_BRIDGE_PORT_FDB_LEARNING_MODE_DISABLE,

    /** Hardware learning. Learn source MAC. Forward based on destination MAC */
    SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW,

    /** Trap packets with unknown source MAC to CPU. Do not learn. Do not forward */
    SAI_BRIDGE_PORT_FDB_LEARNING_MODE_CPU_TRAP,

    /** Trap packets with unknown source MAC to CPU. Do not learn. Forward based on destination MAC */
    SAI_BRIDGE_PORT_FDB_LEARNING_MODE_CPU_LOG,

    /**
     * @brief Notify unknown source MAC using FDB callback.
     *
     * Do not learn in hardware. Do not forward. When a packet from unknown
     * source MAC comes this mode will trigger a new learn notification via FDB
     * callback for the MAC address. This mode will generate only one
     * notification per unknown source MAC to FDB callback.
     */
    SAI_BRIDGE_PORT_FDB_LEARNING_MODE_FDB_NOTIFICATION,

} sai_bridge_port_fdb_learning_mode_t;
```